stressberry/cli.py

```diff
@@ -164,4 +240,32 @@ def _get_parser_plot():
         default=None,
         help="limits for the temperature (default: data limits)",
     )
+    parser.add_argument(
+        "-d",
+        "--dpi",
+        type=int,
+        default=None,
+        help="image resolution in dots per inch when written to file",
+    )
+    parser.add_argument(
+        "-f",
+        "--frequency",
+        help="plot CPU core frequency (single input files only)",
+        action="store_true",
+    )
+    parser.add_argument(
+        "-l",
+        "--freq-lims",
+        type=float,
+        nargs=2,
+        default=None,
+        help="limits for the frequency scale (default: data limits)",
+    )
+    parser.add_argument("--hide-legend", help="do not draw legend", action="store_true")
+    parser.add_argument(
+        "--not-transparent", help="do not draw legend", action="store_true"
```

Comment:
Let's call the variable `args.transparent` and use `store_false`, default `True`, to avoid weird constructions like `not not_transparent`.

Reply:
Agree this is a bit of a mess. However, calling with `--transparent` makes the image opaque. Which looks wrong. The logic could be reversed, but that would require existing users to call with --transparent if they wanted a transparent image. 

Alternatively, the argument could be `opaque` or `solid-background` or similar. Thoughts?